docs/bir-spec/src/main/resources/kaitai/bir.ksy

```diff
@@ -645,7 +645,7 @@ types:
             'type_tag_enum::type_tag_decimal': decimal_constant_info
             'type_tag_enum::type_tag_boolean': boolean_constant_info
             'type_tag_enum::type_tag_nil': nil_constant_info
-            'type_tag_enum::type_tag_map': map_constant_info
+            'type_tag_enum::type_tag_intersection': map_constant_info
```

Comment:
At the moment, if the type tag is intersection, the value is definitely a mapping value, so this will work for the time-being. But ideally we should look at the effective type and select what to do. We may have to change the BIR instead to use the effective type when writing the constant.